charts/pytorchjob/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for PyTorchJob
+name: pytorchjob
+version: 0.0.1
```

Comment:
the version(0.0.1) is not matched the version(0.1.0) of the CHANGELOG.md?

Reply:
ok